addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,28 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        margin-top: 75px;
+        margin-bottom: -35px;
+        img {
+            height: 185px;
+            width: 275px;
+        }
+        .o_mobile_barcode {
+            opacity: 0;
```

Comment:
I see, it is a bit strange to have useless elements in the view but it works. I would at least do a summary of your trick above o_mobile_barcode then:

// In order to have the o_mobile_barcode button on both the image and the label,
// We use negative margin at the bottom and 0 opacity (since not needed in the view)
.o_mobile_barcode {
...
}